app/controllers/get-users.js

```diff
@@ -0,0 +1,17 @@
+const { User } = require('../models');
+const logger = require('../logger');
+
+exports.getUsers = async (req, res, next) => {
+  const { offset, limit } = req.query;
+  try {
+    const { rows } = await User.findAndCountAll({
+      offset,
+      limit
+    });
```

Comment:
It's better to have this operations in a service function

Reply:
Done